Week-3/refactoring_stack.js

```diff
@@ -0,0 +1,22 @@
+function createStack() {
+  return {
+    push(item) {
+      obj.items.push(item);
+    },
+    pop() {
+      obj.items.pop();
+    },
+  };
+}
+
+var obj = {
```

Comment:
@rajeshvlega9 please move this declaration to the function. Because i can now access as obj.items right. We need an abstraction here to hide out the details of createStack implementation